src/olympia/addons/forms.py

```diff
@@ -417,7 +417,7 @@ def __init__(self, *args, **kwargs):
         cats = sorted(cats, key=lambda x: x.name)
         self.fields['category'].choices = [(c.id, c.name) for c in cats]
 
-        for field in ('header', 'footer'):
+        for field in ('header'):
```

Comment:
Same here, the for loop iterates over `h`, `e`, `a`, `d`, etc. Needs to be a proper tuple again: `('header', )`